README.md

```diff
@@ -0,0 +1,60 @@
+# Reyan
+ Reyan is a free modern open-source application based on the latest feature of the Android/IOS development world which'll let you recite the holy quran easier than ever!
+
+## Tools And Technolagies
+* programming language: [Kotlin](https://kotlinlang.org/docs/reference/android-overview.html)
+* Architecture: [MVI](http://hannesdorfmann.com/android/model-view-intent)
+* MultiThreading: [Coroutines](https://kotlinlang.org/docs/reference/coroutines/coroutines-guide.html)
+* Android DI: [Dagger2](https://dagger.dev/users-guide)
+* General networking: [ktor.http](https://ktor.io/clients/http-client/engines.html)
+* Android networking: [OkHttp engine](https://square.github.io/okhttp)
+* Testing: [JUnit](https://junit.org/junit5), [Truth](https://truth.dev/), [AndroidX Test](https://developer.android.com/training/testing/set-up-project), [Mock](https://mockk.io/ANDROID.html)
+* Misc: [ExoPlayer](https://exoplayer.dev)
+* [Kotlin multi-platform](https://kotlinlang.org/docs/reference/multiplatform.html)
+
+## About
+  Reyan means "**Gift of God**" and Reyan applicaton, represents the most valuable gift of God which is The Holy Quran.
+  
+  Our goal is to make, reading the holy quran easier than ever and encourage our user to read more.
+ 
+# Contributions
+* All tasks and goals are listed in Trello board: [Quran Trello board](https://trello.com/b/pSsteeb5/quran)
+* Bug fixes and PR requests are highly appreciated and you're more than welcome to send us your feedbacks
+
+
+## Installing & Running
+
+ Download the Reyan source code from https://github.com/islamversity/Reyan (git clone https://github.com/islamversity/Reyan.git)
+
+ **Create a local.properties in root folder and add a field for these keys:**
+  * sdk.dir
+  * debugKeyStoreAddress
+  * debugKeyStorePass
+  * debugKeyAlias
+  * debugKeyPass
+ 
+ 
+For running Reyan you will be needing 
+* Android Studio version 4.0 or higher
+* Android gradle plugin version 4.0.1
+* Gradle version 6.3
+* To generate google-service.json and copying it to the root project/app

```

Comment:
the first "to" is incorrect
"to generate google-services.json" -> "Generate google-services.json"